index.html

```diff
@@ -14,25 +14,177 @@
             <h2><a href="#">My Logo</a></h2>
         </div>
         <nav class="open_mobile_menu_icon">
-            <img src="Icons/Menu.svg" alt="">
+            <img src="./Icons/Menu.svg" alt="">
         </nav>
     </header>
-    <section class="headline">
-        <div class="welcome_text">
-            <h1>Hey There.<br> I'm Christian.<br><span>I’m a software developer</h1></span>  
-        </div>
-        <div class="introduction_text">
-            <p class="">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-        </div>
-        <div class="social_media_icons">
-            <ul>
-                <li><a href="https://twitter.com/Momo_yde"><img src="Icons/twitter.svg" alt=""></a></li>
-                <li><a href="https://www.linkedin.com/in/christian-momo/"><img src="Icons/Linkedin.svg" alt=""></a></li>
-                <li><a href=""><img src="Icons/medium.svg" alt=""></a></li>
-                <li><a href="https://github.com/Momo-87"><img src="Icons/github.svg" alt=""></a></li>
-                <li><a href=""><img src="Icons/angellist.svg" alt=""></a></li>
-            </ul>
-        </div>
-    </section>
+    <main>
+        <section class="headline">
+            <div class="welcome_text">
+                <h1>Hey There.<br> I'm Christian.<br><span>I’m a software developer</h1></span>  
+            </div>
+            <div class="introduction_text">
+                <p class="">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="social_media_icons">
+                <ul>
+                    <li><a href="https://twitter.com/Momo_yde"><img src="Icons/twitter.svg" alt=""></a></li>
+                    <li><a href="https://www.linkedin.com/in/christian-momo/"><img src="Icons/Linkedin.svg" alt=""></a></li>
+                    <li><a href=""><img src="Icons/medium.svg" alt=""></a></li>
+                    <li><a href="https://github.com/Momo-87"><img src="Icons/github.svg" alt=""></a></li>
+                    <li><a href=""><img src="Icons/angellist.svg" alt=""></a></li>
+                </ul>
+            </div>
+            <div class="action">
+                <a href="#" class="icon"><img src="./Icons/Down.svg" alt=""></a>
+            </div>
+        </section>
+        <!--Headline section end-->
+        <section class="recent_works">
+            <div class="recent_work_primary_text">My Recent<br> Works</div>
```

Comment:
- [ ] Please the title (e.g., "My Recent Works") should use a header tag `hx` since its better for SEO to use hx for headings